app/src/main/java/com/example/androidreduxsample/redux/Store.kt

```diff
@@ -20,15 +25,15 @@ class Store @Inject constructor(
         )
     )
 
-    val appStateFlow: StateFlow<AppState> = _appStateFlow.asStateFlow()
+    val appStateFlow: StateFlow<AppState>
+        get() = _appStateFlow.asStateFlow()
```

Comment:
does this makes any difference from the existing ?

Reply:
its a computed property, means I am not storing it in variable but getting it on-the-fly.